src/lib/core/say.c

```diff
@@ -153,10 +148,13 @@ static const char *level_strs[] = {
 	[S_DEBUG] = "DEBUG",
 };
 
-static const char*
-level_to_string(int level)
+static_assert(lengthof(level_strs) == say_level_MAX);
```

Comment:
This doesn't compile on OS X, because there `static_assert` must have a message.

Reply:
Fixed to:
```c
static_assert(lengthof(level_strs) == say_level_MAX,
              "level_strs is not defined for one of log levels");
```
here and in the other case.